prow/cmd/mkpj/main.go

```diff
@@ -135,6 +135,10 @@ func (o *options) Validate() error {
 		return errors.New("required flag --config-path was unset")
 	}
 
+	if err := o.github.Validate(false); err != nil {
```

Comment:
The dryRun argument for Validate doesn't make much sense and I think it can be removed. Wdyt?

Reply:
In a follow-up! Totally, if it make no sense.